vars/cloudFoundryDeploy.groovy

```diff
@@ -462,7 +465,7 @@ def deployCfNative(config) {
         config.deployOptions,
         config.cloudFoundry.manifest ? "-f '${config.cloudFoundry.manifest}'" : null,
         config.deployDockerImage ? "--docker-image ${config.deployDockerImage}" : null,
-        binding.hasVariable("dockerUsername") ? "--docker-username ${dockerUsername}}" : null,
+        config.dockerUsername ? "--docker-username ${dockerUsername}" : null,
```

Comment:
Just a little bit background why I chose to use `binding.hasVariable(...)`: 

I had issues to access the variables, if they weren't defined (hence, the `dockerUsername` config parameter was not set). But the tests are running and it seems to work. Maybe the issue was just connected to my environment. 

The change looks good to me.